contrib/kind-common

```diff
@@ -122,12 +122,13 @@ install_ingress() {
 
 METALLB_DIR="/tmp/metallb"
 install_metallb() {
+  local metallb_version=v0.14.8
```

Comment:
hmm why not 0.14.18 the latest version?
any specific reason for choosing .8?

Reply:
The .8 tag corresponds to .18 release, don't know why though.

Reply:
@tssurya -> https://github.com/metallb/metallb/releases/tag/v0.14.8